Topic 0/src/point2/Demo.java

```diff
@@ -0,0 +1,26 @@
+package point2;
+
+import java.util.ArrayList;
+
+
+public class Demo {
+
+	public static void main(String[] args) {
+		
+		ArrayList<Room> aux= new ArrayList<Room>();
```

Comment:
[Medium] When you declare a variable, it is better to refer Interfaces or general classes when that it's possible. In this case for example,

```java
   List<Room> rooms = new ArrayList<Room>();
```

This will allow you to change the list structure if you needed, without any other changes in our code.